components/network/openvpn/Makefile

```diff
@@ -37,7 +39,6 @@ CONFIGURE_OPTIONS += CPPFLAGS="$(CPPFLAGS)"
 CONFIGURE_OPTIONS += GREP=/usr/gnu/bin/grep
 CONFIGURE_OPTIONS += --disable-static
 CONFIGURE_OPTIONS += --enable-shared
-CONFIGURE_OPTIONS += --enable-password-save
```

Comment:
Just in case - what's the deal with this feature? Was it useful? was it removed in upstream? Can we have existing deployments wanting/relying on it? :)